### DISCUSSION TITLE
How to change the default layout file location

### DISCUSSION THREAD

**Discussion by edthetechie** [wheels-dev/wheels#1238] [Q&A]
Hey guys, I want to organize my layouts a bit better so I put them in a folder but Wheels still wants a layout.cfm in the views root. Is there a way to change that? I kludged it by adding a layout.cfm with a cfinclude in it but then it occurred to me that this is not a very "wheelsy" way of doing it. I checked the docs but couldn't find anything. Does that mean it's not possible and my kludge is fine or is there a way to do it and I just missed it in the docs?

**Comment by neokoenig** [comment]
Haven't tried it for a while, but could you not do this in your main Controller.cfc ? :
```
config(){
usesLayout(template="/layouts/default");
}
```

**Reply by edthetechie** [reply]
Tried that, it didn't work. No biggie, this isn't super important, my solutions works well enough. Thanks anyway :-)

**Comment by chapmandu** [comment]
I have my layouts in `/views/layouts/`

Then I have my base controller's `config()` call `usesLayout("setLayout");` `setLayout` is a function that can dynamically change layouts..

```
private string function setLayout() {
	if (params.layout == "false") {
		return false;
	} else if (params.layout == "fancy") {
		return "/layouts/fancy";
	}
	return "/layouts/standard";
}
```

Tom's (simpler) approach will also work if your layout is located in `/views/layouts/default.cfm`

**Reply by edthetechie** [reply]
That's a really good idea, thanks Adam 👍

**Comment by zainforbjs** [comment]
@edthetechie If this discussion is answered, then can you mark it as answered?
